.github/workflows/python-pipeline.yml

```diff
@@ -0,0 +1,36 @@
+name: Python Container for Text Extraction
```

Comment:
Nice to have automated pip installs that actually work (unix superiority). However, due to running/linting with Flake8 after line `28` we'll no doubt run into errors with most of the Python files in the repository. For this reason, I'll be fixing this by reducing the majority of the flask boilerplate code to an SPA that calls the full flask app/notebook at https://github.com/signal-k/sytizen/issues/18 / https://github.com/signal-k/polygon/issues/24)